qemu/tests/cfg/power_htm.cfg

```diff
@@ -0,0 +1,26 @@
+- power_htm:
+    type = power_htm
+    only ppc64le, ppc64
+    machine_type_extra_params = "cap-htm=on"
+    start_vm = yes
+    unit_test = no
+    htm_dir = /tmp/htm
+    get_htm_dir = "rm -rf ${htm_dir}; mkdir ${htm_dir}; cd ${htm_dir}"
+    variants:
+        - unit_test:
+            start_vm = no
+            unit_test = yes
+            get_htm_dir +="; git clone https://gitlab.com/kvm-unit-tests/kvm-unit-tests"
+            get_unit_dir = ${htm_dir}/kvm-unit-tests
+            compile_htm = "cd ${get_unit_dir}; ./configure --endian=little"
+            compile_htm += "; make standalone"
+            compile_htm += "; export ACCEL=kvm,kvm-type=HV"
+            run_htm_test = "cd ${get_unit_dir}; for test in tests/*; do echo yes | $test; done"
+        - guest_test:
+            depends_pkgs = "glibc-static"
+            check_htm_env = "cat /sys/firmware/devicetree/base/ibm"
+            check_htm_env+= ",opal/fw-features/tm-suspend-mode/enabled"
+            # please configure your url
+            download_htm_demo = < download_htm_demo >
```

Comment:
Also comment on it since it's not a valid link.

Reply:
will update it

Reply:
how about give an test.error message but keep it ?